src/lighthouse.php

```diff
@@ -453,4 +453,24 @@
          */
         'entities_resolver_namespace' => 'App\\GraphQL\\Entities',
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Tracing
+    |--------------------------------------------------------------------------
+    |
+    | Configuration for tracing support.
+    |
+    */
+
+    'tracing' => [
+        /*
+         * Driver used for tracing.
+         *
+         * Available options: '\Nuwave\Lighthouse\Tracing\ApolloTracing\ApolloTracing::class', '\Nuwave\Lighthouse\Tracing\FederatedTracing\FederatedTracing::class'
+         *
+         * Will be removed in Lighthouse v7 and implicitly default to 'Nuwave\Lighthouse\Tracing\FederatedTracing\FederatedTracing::class'.
```

Comment:
I guess we can still allow overwriting the binding of the scoped singleton for customization then?

Reply:
Yes we can simply change the default in v7 but keeping the driver structure so it allows customization